src/model/transform-trace-data.js

```diff
@@ -25,7 +25,12 @@ type SpanWithProcess = SpanData & { process: Process };
  * NOTE: Mutates `data` - Transform the HTTP response data into the form the app
  * generally requires.
  */
-export default function transfromTraceData(data: TraceData & { spans: SpanWithProcess[] }): Trace {
+export default function transfromTraceData(data: TraceData & { spans: SpanWithProcess[] }): ?Trace {
+  const traceID = data.traceID.toLowerCase();
+  if (!traceID) {
```

Comment:
this looks like a check for traceID==null, shouldn't it be before L29?

Reply:
Checking for empty string, but to be cautious will adjust.